### PR TITLE
Fix dns benchmark failing after python 2->3 migration

### PR DESCRIPTION
**Pull request by mm4tt**
Ref. https://github.com/kubernetes/perf-tests/issues/1586

### CODE REVIEW

File: dns/py/runner.py [hunk 1]
```diff
@@ -161,8 +162,6 @@ def _kubectl(self, stdin, *args):
     """
     cmdline = [self.args.kubectl_exec] + list(args)
     _log.debug('kubectl %s', cmdline)
-    if stdin:
-      _log.debug('kubectl stdin\n%s', add_prefix('in  | ', stdin))
```
Comment (mm4tt) [inline dns/py/runner.py:165]: I removed this logging as it was causing more harm than good. We're logging stdin here, which can be anything, e.g. a tar archive...

**Comment by wojtek-t** [comment]
/lgtm

It seems the tests are passing now, which is great.

/hold
Waiting for Pavithra.

**Comment by prameshj** [comment]
/hold cancel
/lgtm

Thanks Matt!

**Pull request merged**